$NetBSD: patch-Makefile.am,v 1.1 2015/12/31 07:07:16 ryoon Exp $

At least on SunOS, lang/nawk (nawk-20050424nb3) insists on
a space between -f and the program file name, by penalty of
"/opt/local/bin/nawk: no program filename"

--- Makefile.am.orig	2015-10-18 17:10:43.000000000 +0000
+++ Makefile.am
@@ -242,7 +242,7 @@ install-exec-hook:
 			>$(srcdir)/tmux.1.mdoc; \
 	else \
 		sed -e "s|@SYSCONFDIR@|$(sysconfdir)|g" $(srcdir)/tmux.1| \
-			$(AWK) -f$(srcdir)/mdoc2man.awk >$(srcdir)/tmux.1.man; \
+			$(AWK) -f $(srcdir)/mdoc2man.awk >$(srcdir)/tmux.1.man; \
 	fi
 	$(mkdir_p) $(DESTDIR)$(mandir)/man1
 	$(INSTALL_DATA) $(srcdir)/tmux.1.@MANFORMAT@ \
